task_list/app/views/tasks/show.html.erb

```diff
@@ -20,11 +23,15 @@
       </td>
       <td>
         <% if @task.completed_at == nil %>
-          <input type="checkbox" name="completed[]" value="incomplete">
+              <!-- <% puts "No" %> -->
+          <!-- <input type="checkbox" name="completed[]" value="incomplete"> -->
```

Comment:
Although this isn't being used anymore, if you have a form where you know there will be just one checkbox for a particular form parameter (such as `completed`), you don't need to use the array syntax in the input name.